setup.py

```diff
@@ -75,12 +75,26 @@
 from setuptools import find_packages
 
 
+def find_version(*file_paths):
+    """
+    https://github.com/pypa/pip/blob/404838abcca467648180b358598c597b74d568c9/setup.py#L18-L28
+    """
+
+    version_file = read(*file_paths)
+    version_match = re.search(r"^__version__ = ['\"]([^'\"]*)['\"]",
+                              version_file, re.M)
+    if version_match:
+        return version_match.group(1)
+    raise RuntimeError("Unable to find version string.")
+
+
 with open('README.md') as file_object:
   long_description = file_object.read()
 
+
 setup(
   name = 'tuf',
-  version = '0.11.1',
+  version = find_version("tuf", "__init__.py"),
```

Comment:
This isn't going to work: read isn't defined. This is code from [here](https://packaging.python.org/guides/single-sourcing-package-version/) and is a little more complicated than is ideal. It'll also match comment lines if they exist. Single-sourcing version number isn't necessary for this pull request, but if I was going to do it, I'd probably add a VERSION file and have `tuf/__init__.py` and `setup.py` each read that in. There could be problems with that, too. I'm going to punt on this and keep the version in two places and we can fix that less urgently. (Also, the user agent reporting a version seems less critical in any case than the rest of the PR.)